industrial_ci/src/builders/catkin_tools_devel.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright (c) 2021, Mathias Lüdtke
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# shellcheck source=industrial_ci/src/builders/catkin_tools.sh
+source "${ICI_SRC_PATH}/builders/catkin_tools.sh"
+
+ici_warn "BUILDER=catkin_tools_devel should only be used in addition to the other non-devel builders"
+
+function ici_extend_space {
+    echo "$1/devel"
+}
```

Comment:
Shouldn't you drop the `--install` flag here as well?

Reply:
Again, install should be tested.

Reply:
I agree here if you want to test `--install`, but not use it.